src/CohortList.js

```diff
@@ -0,0 +1,120 @@
+import Cohort from './Cohort.js'
+
+class CohortList {
+  constructor() {
+    this.list = []
+    this.studentsList = []
+  }
+
+  // Create Cohort
+  createCohort(name) {
```

Comment:
This comment is like writing "Pen" on a label and then putting the label on a pen. Your function describes itself well, no need for comments :)